crates/re_viewer_context/src/annotations.rs

```diff
@@ -23,10 +23,8 @@ pub struct Annotations {
 impl Annotations {
     pub fn try_from_view(view: &ArchetypeView<AnnotationContext>) -> Option<Self> {
         re_tracing::profile_function!();
-        // TODO(jleibs): Mono helpers for ArchetypeView.
-        view.iter_required_component::<re_types::components::AnnotationContext>()
+        view.required_mono_component::<re_types::components::AnnotationContext>()
```

Comment:
We require a mono component but then we just return `None` if it's not there? Shouldn't that do an optional query to begin with?

Reply:
Good catch